Dockerfile

```
@@ -76,17 +76,19 @@ EOF
 # runtime stage: produces final image that will be executed
 FROM --platform=linux/riscv64 riscv64/ubuntu:22.04
 
-LABEL io.sunodo.sdk_version=0.2.0-sandboxing
+LABEL io.sunodo.sdk_version=0.4.0
 LABEL io.cartesi.rollups.ram_size=128Mi
 LABEL io.cartesi.rollups.data_size=128Mb
 
-ARG MACHINE_EMULATOR_TOOLS_VERSION=0.12.0
+ARG MACHINE_EMULATOR_TOOLS_VERSION=0.14.1
+ARG MACHINE_EMULATOR_TOOLS_DEB=machine-emulator-tools-v${MACHINE_EMULATOR_TOOLS_VERSION}.deb
 RUN <<EOF
```

Since we're already changing the Dockerfile, I suggest making some changes to make it more consistent.

Adding `set -e` on every `RUN <<EOF`.

And add `ARG DEBIAN_FRONTEND=noninteractive` if there's an `apt-get` command being used.

```suggestion
ARG DEBIAN_FRONTEND=noninteractive
RUN <<EOF
set -e
```

Also, put each package in a single line and sorted.